docker/deploy.sh

```diff
@@ -2,6 +2,15 @@
 
 set -euo pipefail
 
+function autodeploy {
+    # Notifying webhook
+    curl -s --output /dev/null --write-out "%{http_code}" \
+    -H "Content-Type: application/json" \
```

Comment:
nit: maybe indent params here so its easy to identify that they pertain to the `curl` call